.github/workflows/check-policy-signatures.yml

```diff
@@ -0,0 +1,38 @@
+# This action is ran manually or via the dedicated `workflow` API of GitHub.
+#
+# The action receives as input the name of the GitHub repository that
+# holds the policy, plus the tag associate with the release event of
+# that policy.
+#
+# The action downloads the `hub.yml` file from the repository mentioned
+# by the input variables.
+# The action then replaces the special `__TAG__` string inside of the
+# `hub.json` file with the value of the `tag` it received at invocation
+# time.
+#
+# The action uses yq to convert the incoming yml data to json format
+# and saves the output to `web/policies/<org name>:<policy name>.json`
+#
+# Finally, the action creates a PR against this repository with the
+# changes done to the file.
```

Comment:
This whole docs has been copied from another action, can you update it to reflect what is being done by this action?